tests/pass-dep/tokio/sleep.rs

```diff
@@ -1,5 +1,4 @@
-//@compile-flags: -Zmiri-permissive-provenance -Zmiri-backtrace=full
-//@only-target-x86_64-unknown-linux: support for tokio only on linux and x86
+//@only-target-linux: support for tokio only on linux
```

Comment:
```suggestion
//@only-target-linux: We only support tokio on Linux
```
same for the other files